### PR TITLE
Basic Analysis

### PR DESCRIPTION
**Pull request by lukeraus**
I Want to merge my lyrics branch to the milestone 1

### CODE REVIEW

File: analysis/watason_personality.js
Comment (tacastillo) [inline analysis/watason_personality.js]: I'll approve it, and if you don't want to change it right now, i can do it later, but the file is called `watason_personality.js` as a heads up.

**Review by tacastillo — APPROVED** [review]

File: analysis/watason_personality.js
Comment (lukeraus) [reply analysis/watason_personality.js]: Whoops good catch I'll update that before merging

**Pull request merged**
